scikeras/wrappers.py

```diff
@@ -987,12 +980,6 @@ class KerasRegressor(BaseWrapper):
         }
     )
 
-    def _validate_data(self, X, y=None, reset=True):
-        """Convert y to float, regressors cannot accept int."""
-        if y is not None:
-            y = check_array(y, ensure_2d=False)
-        return super()._validate_data(X=X, y=y, reset=reset)
-
```

Comment:
This can actually be removed now, thanks to the changes in `BaseWrapper._validate_data`!